samples/Docker/compose.yaml

```diff
@@ -0,0 +1,51 @@
+# Tell us about your experience using dotnet monitor: https://aka.ms/dotnet-monitor-survey
+services:
+  app:
+    image: mcr.microsoft.com/dotnet/samples:aspnetapp-chiseled
+    ports:
+      - "8080:8080"
+    environment:
+      - DOTNET_DiagnosticPorts=/diag/dotnet-monitor.sock
+    volumes:
+      - diagvol:/diag
+    deploy:
+      resources:
+        limits:
+          cpus: '0.250'
+          memory: 512M
+
+  monitor:
+    image: mcr.microsoft.com/dotnet/monitor:8
```

Comment:
Any reason we're using 8 for these examples instead of 9?

Reply:
Just copy/paste, we should fix up docs to reference the latest major tag wholesale.